CMakeLists.txt

```diff
@@ -65,6 +57,7 @@ if (BUILD_DEPS)
     set(BUILD_TESTING OFF)
     add_subdirectory(crt/aws-c-common)
     if (UNIX AND NOT APPLE)
+        include(crt/aws-c-cal/cmake/modules/aws-lc.cmake)
         add_subdirectory(crt/s2n)
```

Comment:
just curious, why is it `include(aws-lc.cmake)` but `add_subdirectory(every-other-lib)`?